core/platform/taskqueue/cloud_tasks_emulator.py

```diff
@@ -0,0 +1,237 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""An emulator for that mocks the core.platform.taskqueue API. This emulator
+models the third party library Google Cloud Tasks.
```

Comment:
Please make sure to add a reference to the third-party GitHub lib on which this file is based.